uptane-standard.md

```diff
@@ -555,8 +566,8 @@ The Image repository MAY require authentication for read access.
 
 The Director repository instructs ECUs as to which images should be installed by producing signed metadata on demand. Unlike the Image repository, it is mostly controlled by automated, online processes. It also consults a private inventory database containing information on vehicles, ECUs, and software revisions.
 
-The Directory repository SHALL expose an interface for primaries to upload vehicle version manifests ({{vehicle_version_manifest}}) and download metadata. This interface SHOULD be public.
-The Director MAY encrypt images for ECUs that require it, either by encrypting on-the-fly or by storing encrypted images in the repository.
+The Director repository SHALL expose an interface for primaries to upload vehicle version manifests ({{vehicle_version_manifest}}) and download metadata. This interface SHOULD be public.
+The Director MAY encrypt images for ECUs that require it, either by encrypting on-the-fly or by storing encrypted images on the repository.
```

Comment:
@tkfu and I went back and forth about whether "in" or "on the repository" was better. To my ears, "in" is the only one that sounds right, and I think I convinced Jon in the end.

Reply:
Fixed.